tests/tools/onert_train/src/onert_train.cc

```diff
@@ -121,45 +121,6 @@ int main(const int argc, char **argv)
     verifyInputTypes();
     verifyOutputTypes();
 
-    auto convertLossType = [](int type) {
-      switch (type)
-      {
-        case 1:
-          return NNFW_TRAIN_LOSS_MEAN_SQUARED_ERROR;
-        case 2:
-          return NNFW_TRAIN_LOSS_CATEGORICAL_CROSSENTROPY;
-        default:
-          std::cerr << "E: not supported loss type" << std::endl;
-          exit(-1);
-      }
-    };
-
-    auto convertLossReductionType = [](int type) {
-      switch (type)
-      {
-        case 1:
-          return NNFW_TRAIN_LOSS_REDUCTION_SUM_OVER_BATCH_SIZE;
-        case 2:
-          return NNFW_TRAIN_LOSS_REDUCTION_SUM;
-        default:
-          std::cerr << "E: not supported loss reduction type" << std::endl;
-          exit(-1);
-      }
-    };
-
-    auto convertOptType = [](int type) {
-      switch (type)
-      {
-        case 1:
-          return NNFW_TRAIN_OPTIMIZER_SGD;
-        case 2:
-          return NNFW_TRAIN_OPTIMIZER_ADAM;
-        default:
-          std::cerr << "E: not supported optimizer type" << std::endl;
-          exit(-1);
-      }
-    };
```

Comment:
## Motivation

I'd like to remove this part from `onert_train`. 
Because this part highly depends on `onert_train/args.h `. 
   * Now, If we update argument list in onert_train/args.h, we have to update this onert_train main function too.